src/node_binding.cc

```diff
@@ -52,6 +58,7 @@
   V(http_parser)                                                               \
   V(inspector)                                                                 \
   V(js_stream)                                                                 \
+  V(js_udp_wrap)                                                               \
```

Comment:
could js udp wrap be a standalone feature?

Reply:
Yes. The drawback is that this is currently only used to enable testing scenarios for the QUIC implementation. If we add it as a separate PR in core, nothing else would use it until QUIC landed.